custom/icds_reports/messages.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals
+from django.utils.translation import ugettext as _
+
+
+def wasting_help_text(beta):
+    if beta:
+        return _(
+            "Percentage of children between 0 - 5 years enrolled for Anganwadi Services with "
+            "weight-for-height below -2 standard deviations of the WHO Child Growth Standards median. "
+            "<br/><br/>"
+            "Severe Acute Malnutrition (SAM) or wasting in children is a symptom of acute "
+            "undernutrition usually as a consequence of insufficient food intake or a high "
+            "incidence of infectious diseases."
+        )
+    return _(
+        "Percentage of children between 6 - 60 months enrolled for Anganwadi Services with weight-for-height "
+        "below -3 standard deviations of the WHO Child Growth Standards median. "
+        "<br/><br/>"
+        "Severe Acute Malnutrition (SAM) or wasting in children is a symptom of acute "
+        "undernutrition usually as a consequence of insufficient food intake or a high "
+        "incidence of infectious diseases."
+    )
+
```

Comment:
I want to move all strings to one file, currently, we have a lot of duplicates. I will be updating this file on the fly.